flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java

```diff
@@ -568,12 +592,7 @@ && isChainableInput(sourceOutEdge, streamGraph)) {
                     final StreamConfig.SourceInputConfig inputConfig =
                             new StreamConfig.SourceInputConfig(sourceOutEdge);
                     final StreamConfig operatorConfig = new StreamConfig(new Configuration());
-                    setVertexConfig(
-                            sourceNodeId,
-                            operatorConfig,
-                            Collections.emptyList(),
-                            Collections.emptyList(),
-                            Collections.emptyMap());
+                    setOperatorConfig(sourceNodeId, operatorConfig, Collections.emptyMap());
```

Comment:
IIUC, `setChainedOutputs(emptyList)` would be called here previously, but is no longer called after this change?

Reply:
Yes. I will modify it to the same logic as before